### ISSUE TITLE
Server-Side Request Forgery in axios

### ISSUE THREAD

**Issue by mathiasschopmans** [ActiveCampaign/postmark.js#160]
### npm audit report
```
axios  >=1.3.2
Severity: high
Server-Side Request Forgery in axios - https://github.com/advisories/GHSA-8hc4-vh64-cxmj
```

### Fix: 
Update axios to 1.7.4

**Comment by mathiasschopmans** [comment]
Temporary fix via `package.json`s overrides in project:

```
"overrides": {
    "postmark": {
      "axios": "^1.7.4"
    }
}
```

**Comment by ibalosh** [comment]
Thanks for letting us know, this was resolved in version 4.0.5
